lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1933,6 +1936,13 @@ def increment_successful_requests_count
       @successful_requests_count.increment(grpc: @use_grpc, code: @ok_code)
     end
 
+    # Increment the metric for the number of partially successful requests.
+    def increment_partially_successful_requests_count(code)
+      return unless @partially_successful_requests_count
+      @partially_successful_requests_count.increment(grpc: @use_grpc,
```

Comment:
Will @use_grpc always be false for now while we only invoke this function in the non-grpc block? If so, should we hard code it, or at least parameterize it to the calling function to prevent depending on the class variable?

Reply:
There is a coming PR to add partial success for gRPC. So soon this method will be invoked by both paths.

Reply:
For reference, that PR is https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/174.